src/pages/Main/MainFlexSliderFetch.js

```diff
@@ -2,20 +2,20 @@ import { useState, useEffect } from 'react';
 
 import { FlexibleSlider } from '../../components/FlexibleSlider/FlexibleSlider';
 
-export default function MainFlexSliderFetch({ sectionArea, selectedIcon }) {
+export default function MainFlexSlider({ sectionArea, selectedIcon }) {
   const [products, setProducts] = useState([]);
-  const [isProductFetched, setIsProductFetched] = useState(false);
+  const [isProducted, setIsProducted] = useState(false);
```

Comment:
```suggestion
  const [isLoaded, setIsLoaded] = useState(false);
```
쓰임새를 보니 이름이 그냥 로딩되었는지 여부를 표현하면 될거 같습니다

Reply:
리뷰 사항 반영했습니다!